modules/client/templates/scripts/forseti-client/forseti_client_startup_script.sh.tpl

```diff
@@ -32,13 +32,13 @@ git checkout ${forseti_version}
 sudo apt-get install -y $(cat install/dependencies/apt_packages.txt | grep -v "#" | xargs)
 
 # Forseti dependencies
-pip install --upgrade pip==9.0.3
-pip install -q --upgrade setuptools wheel
-pip install -q --upgrade -r requirements.txt
+pip3 install --upgrade pip==19.1.1
```

Comment:
Can you revert back to 9.0.3 for now? This breaks some of the code as the pip python library changed.

Reply:
Changed.